NAMESPACE

```diff
@@ -20,6 +21,7 @@ export(edit_r_environ)
 export(edit_r_makevars)
 export(edit_r_profile)
 export(edit_rstudio_snippets)
+export(edit_workflow)
```

Comment:
The name seems to have the wrong implied scope to me. It feels very global, whereas I'd like to find a way to suggest the "GitHub actions" scope.